### PR TITLE
Set 'no rows' height

### PR DESCRIPTION
**Pull request by ronnywoods**
My last PR removed superfluous table padding, but this created an issue when table doesn't have any data (entire table is collapsed and there's no room for 'no rows to show' message).  This PR fixes this by setting a table height of 200px whenever there is no data to display.

Diff will also include changes from my last yet-to-be-merged PR.

### CODE REVIEW

File: components/grid/base-table.jsx [hunk 1]
```diff
@@ -168,6 +183,7 @@ export function BaseTable({
 						<AgGridColumn
 							{...columnProps}
 							key={fieldName}
+							colId={fieldName}
```
Comment (korbinancell) [inline components/grid/base-table.jsx]: Looks like the changes for the previous pr ended up here. If you fix the colId here then we can just close the previous one,

File: components/grid/base-table.jsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import * as Styled from './styled';
 
 const defaultRowHeight = 45;
 const headerHeight = defaultRowHeight - 5;
+const noRowsTableHeight = 200;
```
Comment (korbinancell) [inline components/grid/base-table.jsx]: I wonder if you can localize the `No Rows Found` text or whatever it says. Probably not an issue now until we figure out how we want to localize.

File: components/grid/base-table.jsx (hunk 1 above)
Comment (ronnywoods) [reply components/grid/base-table.jsx]: Just fixed in other PR but I'll go ahead and fix here as well.

File: components/grid/base-table.jsx (hunk 2 above)
Comment (ronnywoods) [reply components/grid/base-table.jsx]: Yeah, it seemed like this text was buried in some grid settings somewhere...In our project I'm localizing within each `cellComponent` but we're only gonna support english initially anyway, so not too worried about 'No Rows Found' text for now.

**Pull request merged**